openmc/data/grid.py

```diff
@@ -0,0 +1,114 @@
+import numpy as np
+
+
+def linearize(x, f, tolerance=0.001):
+    """Return a tabulated representation of a function of one variable.
```

Comment:
This comment reads slightly awkward to me. How about "Returns a tabulated representation of a one-variable function"